packages/ses/src/scope-handler.js

```diff
@@ -174,9 +175,10 @@ export const createScopeHandler = (
     getOwnPropertyDescriptor(_target, prop) {
       // Coerce with `String` in case prop is a symbol.
       const quotedProp = q(String(prop));
+      // eslint-disable-next-line @endo/no-polymorphic-call
       console.warn(
         `getOwnPropertyDescriptor trap on scopeHandler for ${quotedProp}`,
-        new Error().stack,
+        new FERAL_ERROR().stack,
```

Comment:
Any error would do equally well here.
```suggestion
        new TypeError().stack,
```